features/step_definitions/fragment.rb

```diff
@@ -0,0 +1,62 @@
+require 'capybara/cucumber'
```

Comment:
There is one step definition file per Web page.
Since there is no  "fragment" page, this file should not exist.
Please move the needed steps (and only them) into existing files.